setup.py

```diff
@@ -70,7 +70,8 @@
         "sphinx_design",
         "sphinx-autodoc-typehints",
         'numpydoc',
-        'pydata_sphinx_theme'
+        'pydata_sphinx_theme',
+        'nbsphinx',
```

Comment:
```suggestion
        'pydata_sphinx_theme',
        'nbsphinx',
        'pandoc'
```
When I checkout your branch and try to `make html`, i get the following error: 
```
reading sources... [  6%] examples/dispatch_tutorial
====================== slowest reading durations =======================
0.011 contrib

Notebook error:
PandocMissing in examples/dispatch_tutorial.ipynb:
Pandoc wasn't found.
Please check that pandoc is installed:
https://pandoc.org/installing.html
make: *** [Makefile:20: html] Error 2
```

`conda install`ing `pandoc` fixed this issue.

Reply:
If this is the entire problem then hooray! However, the installation instructions for `osier` rely on pip, not conda. Will  putting pandoc in the setup.py file properly install pandoc with `pip install -e .[doc]`?

Reply:
Hmmm I thought i replied to this. odd.

The README instructs users to install `osier` with `pip install osier` or `pip install -e .[docs]` in the top-level directory with `setup.py`. Will `pandoc` be properly installed with `pip`? My impression was that it would not be.

Reply:
I did `pip install .[doc]` for `osier`. I just happened to `conda install` the missing `pandoc` package.

Reply:
huh. I could've sworn this wasn't always a thing. Oh well, thanks!

Reply:
@yardasol 

Adding pandoc here will not work, I think, and may have broken some of my other installations. `pip` can only install Python packages. `conda`, while primarily for Python, can install any package, in theory. In other words, `pip` can't install binaries from compiled languages -- only Python, whereas `conda` does.

I'm going to revert this suggestion.

Reply:
You should absolutely put somewhere that `pandoc` is required to build the documentation correctly, perhaps in the install guide or readme.